packages/expandable-arrow/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@hashicorp/react-expandable-arrow",
+  "version": "0.0.1",
+  "description": "",
+  "author": "HashiCorp",
+  "license": "MPL-2.0",
+  "dependencies": {
+    "classnames": "^2.3.1"
+  },
+  "peerDependencies": {
+    "@hashicorp/mktg-global-styles": ">=3.x",
```

Comment:
Is this needed? I don't see any usage of our global styles.

Reply:
🔪 https://github.com/hashicorp/react-components/pull/798/commits/c32a22aa671a116264738a7090b2f312ef96d7e8